content/docs/1.10.1/important-notes/_index.md

```diff
@@ -38,6 +41,101 @@ For the full release note, see [here](https://github.com/longhorn/longhorn/relea
     - [V2 Data Engine Replica Rebuild QoS](#v2-data-engine-replica-rebuild-qos)
     - [V2 Data Engine Volume Expansion](#v2-data-engine-volume-expansion)
 
+## Warning
+
+While the Longhorn v1.8.x to v1.9.x upgrade process typically migrates resources automatically to the new `v1beta2` version, it is required to perform manual CRD storage version migration before upgrading Longhorn to v1.10.x, to ensure that all the Longhorn custom resources are migrated to `v1beta2`, and to remove the legacy `v1beta1` from the CRD stored version.
```

Comment:
```suggestion
While the Longhorn v1.8.x to v1.9.x upgrade process typically migrates resources automatically to the new v1beta2 version, it is recommended to perform manual CRD storage version migration before upgrading Longhorn to v1.10.x. This is because certain operations such as etcd restore or CRD restore may cause legacy v1beta1 data to remain, and a manual migration ensures that all Longhorn custom resources are properly migrated to `v1beta2`.

In addition, it is required to remove the `v1beta1` from the CRD stored version to complete the migration.
```